source/components/dialog/dialog.ts

```diff
@@ -40,6 +41,10 @@ export class DialogComponent extends FormComponent {
 		this.dialogRoot = dialogRoot;
 	}
 
+	ngOnInit() {
+		this.dialogOpenStatus.emit(false);
```

Comment:
the event emitter is referencing the dialog window itsself. Its a boolean representation of the openness of the window, and its a current status signifying it will and or can change. dialogOpenStatus